core/src/main/scala/chisel3/Data.scala

```diff
@@ -435,27 +435,79 @@ abstract class Data extends HasId with NamedComponent with SourceInfoDoc {
     _direction = Some(actualDirection)
   }
 
+  private[chisel3] def stringAccessor(chiselType: Option[String], vals: Option[String]): String = {
```

Comment:
```suggestion
  private[chisel3] def stringAccessor(chiselType: String): String = {
```
This suggestion is related to comments below, but I don't think the first argument to this method is ever `None` (so no need for `Option`), and I think `vals` is only useful for literals which should be handled outside of this method in my opinion.